README.md

```diff
@@ -32,6 +32,18 @@ status = Timeout::timeout(5) {
 }
 ```
 
+Handling timeout  termination 
+
+```ruby
+begin 
+  status = Timeout::timeout(5) {
+    # Something that should be interrupted if it takes more than 5 seconds...
+  }
+rescue Timeout::Error
+  puts 'The process has taken longer than expected'
+end
+```
```

Comment:
This is all documented on Timeout.timeout: https://docs.ruby-lang.org/en/master/Timeout.html#method-c-timeout

How about adding `# It raises Timeout::Error if it takes longer` after line 31 above?